search.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/tidwall/gjson"
+)
+
+// DefaultResponseBodyTitle defines the default header for the body view
+const DefaultResponseBodyTitle = "Response body (F9)"
+
+// Search performs a search on the request
+// returns title, display, error
+func Search(request *Request, searchString string) (string, []byte, error) {
+	isBinary := (strings.Index(request.ContentType, "text") == -1 &&
+		strings.Index(request.ContentType, "application") == -1)
+
+	if isBinary {
+		return DefaultResponseBodyTitle + " [binary content]",
+			[]byte(hex.Dump(request.RawResponseBody)), nil
+	}
+
+	if strings.Index(request.ContentType, "text") != -1 {
+		return SearchText(request, searchString)
+	}
+
+	if request.ContentType == "application/json" {
```

Comment:
That's fair, but in the meantime this should at least be changed to a `strings.Contains(request.ContentType, "application/json")` so it actually works consistently. The current behavior is really confusing if you're not developing wuzz.